editor/store/selectors.js

```diff
@@ -1744,3 +1744,21 @@ export function getSupportedBlocks( state, uid, globallyEnabledBlockTypes ) {
 export function getEditorSettings( state ) {
 	return state.settings;
 }
+
+/*
+ * Returns the editor settings.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {Object} The editor settings object
+ */
+export function getLocking( state, rootUID ) {
+	if ( ! rootUID ) {
+		return getTemplateLock( state );
+	}
+	const blockListSettings = getBlockListSettings( state, rootUID );
+	if ( ! blockListSettings ) {
+		return;
```

Comment:
Why return `void`? Wouldn't it make sense to at least return `null`? It seems more semantically valid...